tests/migration.go

```diff
@@ -172,3 +172,23 @@ func RunStressTest(vmi *v1.VirtualMachineInstance, vmsize, stressTimeoutSeconds
 		time.Sleep(15 * time.Second)
 	}
 }
+
+func ExpectMigrationPhaseRunning(virtClient kubecli.KubevirtClient, migration *v1.VirtualMachineInstanceMigration, vmi *v1.VirtualMachineInstance, timeout int) {
```

Comment:
I find this one odd.
What I see here is a need to "wait" for that migration to start.
It blocks until it detected that it did or failed. If failed, it should come back with an error.

So at the usage, I would expect something like this:
```
Expect(WaitForMigrationPhase(vmim, v1.MigrationRunning).To(Succeed())
```
And there is no need to use `Eventually` then, just a for-loop with a timeout (`wait.PollImmediate`).

Reply:
Looking at this again the function name is misleading,
named `waitForMigrationRunning` (w/o _phase_). 
It checks that the migration is running by checking
both `Migration` object and the `VMI` migration status.
In order to prevent false positive when the VMI status reports the migration is in final state (completed).
Thus it cannot be generic where you can pass the desired phase.

I will rename it and use `wait.PollImmediate` instead of `Eventually`/`Expect`